src/Helpers/APIHelper.js

```diff
@@ -118,7 +118,12 @@ export function getCveIdsList(apiProps) {
         'advisory_available'
     ];
 
-    let parameterArray = constructParameters(apiProps, parameterNames);
+    const apiPropsWithFlag = {
+        ...apiProps,
+        ...shouldUseHybridSystemFilter ? { affecting: undefined, affectingHostType: apiProps.affecting } : {}
```

Comment:
Maybe we can place this bit of code to change the affecting filter key conditionally inside `constructParameters`. With this, we would make sure the correct filter key is used in all API fetch functions.

Reply:
Do you think it is part of the 'select all CVEs doesn't work' bug ? 
Maybe we can open a ticket for it and add these changes in another PR
What do you think @mkholjuraev ?

Reply:
I believe if we open a ticket, it will not get much attention due to other high-prior tasks. The changes should be very small, all we would need to do is move the code highlighted into `constructParameters `. If you think this does not give enough value, feel free to move on :)

Reply:
I have added your suggestion and added a 'false' default value for this argument (shouldUseHybridSystemFilter) inside constructParameters
So it won't effect other actions, only the one we are changing in this PR (getCveIdsList) 👍🏼